packages/expo-asset/src/Asset.js

```diff
@@ -1,9 +1,9 @@
 import uriparser from 'uri-parser';
 import urljoin from 'url-join';
 import { PixelRatio, Platform } from 'react-native';
-import AssetRegistry from 'react-native/Libraries/Image/AssetRegistry';
 import AssetSourceResolver from 'react-native/Libraries/Image/AssetSourceResolver';
```

Comment:
Nit: order alphabetically here, with uppercase before lowercase (ASCII does this)

```suggestion
import AssetRegistry from 'react-native/Libraries/Image/AssetRegistry';
import AssetSourceResolver from 'react-native/Libraries/Image/AssetSourceResolver';
import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
```